specifyweb/frontend/js_src/lib/components/WorkBench/Template.tsx

```diff
@@ -171,7 +171,7 @@ function WbView({
                 </Button.Small>
                 <Button.Small
                   aria-haspopup="dialog"
-                  className="wb-save"
```

Comment:
The wb-save class name was important here - without it workbench save button won't be clickable anymore (the class name was used for attaching the on click handler)
Careful when removing class names - be sure to look for usages

Reply:
@melton-jason thanks for keeping track!